### ISSUE TITLE
SQLite: document extensions

### ISSUE THREAD

**Issue by yoanmalie** [distantnative/search-for-kirby#26]
Hello,
The plugin work well on localhost but today I tried it on a dev environment and I can't build the index, the shell script tell me :

```
Loading Kirby...
Reindexing using the Search plugin...
Error: could not find driver
```

  - Plugin version: 1.0.0
  - Kirby Version : 3.4.4

What am I missing?

Thanks!

**Comment by yoanmalie** [comment]
It look like it's just the `sqlite3.so` atd the `pdo_sqlite3.so` extensions that I have to enable… So I've enabled them but still get the error 🤔

**Comment by yoanmalie** [comment]
Ok I got it. I was wrong with the `pdo_sqlite3.so` one. Here the two extensions I have to enable:

```
extension=sqlite3.so
extension=pdo_sqlite.so
```


Can we add it to the doc so other peoples not familiar with server configuration like me can do it easier?

Thanks!

**Comment by distantnative** [comment]
✅ Will be included in the docs alongside 1.1.0 release this week.
